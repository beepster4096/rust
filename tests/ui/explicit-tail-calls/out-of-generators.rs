#![feature(explicit_tail_calls)]
#![feature(generators)]

fn main() {
    let _generator = || {
        yield 1;
        become f() //~ error: mismatched function ABIs
    };
}

fn f() {}
