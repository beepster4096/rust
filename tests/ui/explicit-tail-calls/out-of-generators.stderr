error: mismatched function ABIs
  --> $DIR/out-of-generators.rs:7:9
   |
LL |         become f()
   |         ^^^^^^^^^^
   |
   = note: `become` requires caller and callee to have the same ABI
   = note: caller ABI is `"rust-call"`, while callee ABI is `"Rust"`

error: aborting due to previous error

